raiden/tests/scenarios/bf1_basic_functionality.yaml

```diff
@@ -40,15 +38,15 @@ nodes:
     default-reveal-timeout: 20
   node_options:
     0:
-      matrix-server: https://transport01.raiden.network
+      matrix-server: {{ matrix_servers[0] }}
     1:
       matrix-server: https://transport02.raiden.network
     2:
       matrix-server: https://transport03.raiden.network
     3:
       matrix-server: https://transport04.raiden.network
```

Comment:
Shouldn't those also come from the environment?

Reply:
Yes, fixed!